website/docs/developer/development-guide/documentation/documentation-deployment.md

```diff
@@ -16,61 +18,58 @@ The release is automated in Bamboo, but the step must be manually started. If
 you set the `RELEASE_FLAG` to `true` and the build steps passed, you will be 
 able to run the manual "Release" step in Bamboo.
 
-The CI release scripts will create a release based on the release version tag, 
-as well as uploading release artifacts to the Github release for the Terraform 
-modules provided by Cumulus. The Terraform release artifacts include:
-
- * A multi-module Terraform .zip artifact containing filtered copies of the 
-   tf-modules, packages, and tasks directories for use as Terraform module sources.
-
-Just make sure to verify the appropriate .zip files are present on Github after 
-the release process is complete.
-
-### Merge the base branch into develop and master
-
-Merge the version update changes back into develop and master.
-
-If this is the latest version, you can simply create a PR to merge the release 
-branch into develop and master. Note: Do not squash this merge. Doing so will 
-make the "compare" view from step 4 show an incorrect diff, because the tag is 
-linked to a specific commit on the base branch.
+On a successful automated release, the updated documentation is committed to the
+`gh-pages` branch of the [Cumulus ORCA repo](https://github.com/nasa/cumulus-orca).
+The updated documentation should be available on the Cumulus ORCA website at
+https://nasa.github.io/cumulus-orca.
 
 ## Manual Documentation Release on Github
 
 Use the following steps to update Github documentation manually.
```

Comment:
Add - for information please see the [Docusaurus deployment documentation](https://docusaurus.io/docs/deployment)